### DISCUSSION TITLE
So many virus detections on VirusTotal...

### DISCUSSION THREAD

**Discussion by jchai5** [DrDonk/unlocker#96] [General]
Bro, I dropped this zip into VirusTotal and here's what it gave... 

![image](https://github.com/DrDonk/unlocker/assets/86175569/583b78ac-1d51-421b-958c-71a54303747f)

Like come on....

**Comment by DrDonk** [comment]
All false positives due to using Go. There are a lot of articles about this. Try compiling it yourself and you will get the same issue.

**Comment by DrDonk** [comment]
I have documented it here in the wiki https://github.com/DrDonk/unlocker/wiki/False-positive-from-antivirus-software


Link to golang FAQ https://go.dev/doc/faq#virus
